UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/AuthTokenProvider.cs

```diff
@@ -28,6 +33,51 @@ public async Task<string> GetEssTokenNoAuth()
             return EssAccessTokenNoAuth;
         }
 
+        public async Task<string> GetFssToken()
+        {
+            FssAccessToken = await GenerateFssToken(FssAuthConfig.AutoTestClientId, FssAuthConfig.AutoTestClientSecret, FssAccessToken);
+            return FssAccessToken;
+        }
+
+        /// <summary>
+        /// Generate FSS Token
+        /// </summary>
+        /// <param name="ClientId"></param>
+        /// <param name="ClientSecret"></param>
+        /// <param name="Token"></param>
+        /// <returns></returns>
+
+        private static async Task<string> GenerateFssToken(string ClientId, string ClientSecret, string Token)
+        {
+            string[] scopes = new string[] { $"{FssAuthConfig.FssClientId}/.default" };
+            if (Token == null)
+            {
+                if (EssauthConfig.IsRunningOnLocalMachine)
```

Comment:
Kindly change this to if (FssAuthConfig.IsRunningOnLocalMachine)

Reply:
Resolved